contracts/MyMultiSig.cdc

```diff
@@ -266,6 +266,10 @@ pub contract MyMultiSig {
         }
 
         pub fun addSigner(signer: Address) {
+            pre {
+                signer != 0x0 : "The signer address should be valid"
```

Comment:
can you also add the following check:
```
getAccount(signer) != nil
```

Reply:
As per discussion, updated this to 
`getAccount(signer).keys.get(keyIndex: 0)`
because it will throw an error if the account doesn't exist